### PR TITLE
600 design changes certslabelsletters

### PR DESCRIPTION
**Pull request by slkendell-mt**
Design & Data changes from testing done by BRT

### CODE REVIEW

File: backoffice/src/views/exports/certificates/Certificate.tsx [hunk 1]
```diff
@@ -182,9 +187,32 @@ const MaritimeUse: FunctionComponent<UseProps> = ({
...
+            classes="half"
+            title="Winffarm Location"
+            value={use.windfarmLocation}
+          />
+        )}
+
+        {use.rigPlatformLocation && (
+          <CertificateField
+            classes="half"
+            title="Winffarm Location"
```
Comment (Evie-Skinner18) [inline backoffice/src/views/exports/certificates/Certificate.tsx:211]: spelling mistake

File: backoffice/src/views/exports/certificates/Certificate.tsx [hunk 2]
```diff
@@ -328,6 +375,34 @@ const LandUse: FunctionComponent<UseProps> = ({
...
+        <CertificateField
+          classes="half"
+          title="Beacon Position"
+          value={use.beaconPosition}
+        />
+
+        {use.windfarmLocation && (
+          <CertificateField
+            classes="half"
+            title="Winffarm Location"
```
Comment (Evie-Skinner18) [inline backoffice/src/views/exports/certificates/Certificate.tsx:393]: spelling mistake

File: service/src/main/java/uk/gov/mca/beacons/api/beaconuse/domain/BeaconUse.java [hunk 3]
```diff
@@ -329,4 +328,14 @@ public Map<String, String> getCommunicationTypes() {
...
     return communicationTypes;
   }
+
+  public String getUseType() {
+    String activityName = getActivity() == Activity.OTHER
+      ? BeaconsStringUtils.valueOrEmpty(otherActivity)
+      : BeaconsStringUtils.valueOrEmpty(
+        BeaconsStringUtils.enumAsString(activity)
+      );
+
```
Comment (Evie-Skinner18) [inline service/src/main/java/uk/gov/mca/beacons/api/beaconuse/domain/BeaconUse.java:338]: Lush

Comment (Evie-Skinner18) [reply service/src/main/java/uk/gov/mca/beacons/api/beaconuse/domain/BeaconUse.java:338]: Love seeing logic inside domain objects

File: service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java [hunk 4]
```diff
@@ -67,10 +68,12 @@ public LabelDTO toLegacyLabelDTO(LegacyBeacon beacon) {
...
       .mcaContactNumber("+44 (0)1326 317575")
       .beaconUse(
-        mainUse != null ? StringUtils.valueOrEmpty(mainUse.getName()) : ""
+        mainUse != null
+          ? BeaconsStringUtils.valueOrEmpty(mainUse.getName())
+          : ""
       )
       .hexId(beacon.getHexId())
-      .coding(StringUtils.valueOrEmpty(beaconData.getCoding()))
+      .coding(BeaconsStringUtils.valueOrEmpty(beaconData.getCoding()))
```
Comment (Evie-Skinner18) [inline service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java:76]: good idea to make our utils Beacons-specific

File: service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java [hunk 5]
```diff
@@ -155,19 +163,33 @@ private BeaconExportMaritimeUseDTO toMaritimeUse(BeaconUse use) {
     return BeaconExportMaritimeUseDTO
       .builder()
       .environment(use.getEnvironment().toString())
+      .typeOfUse(use.getUseType())
+      .beaconPosition(use.getBeaconPosition())
+      .beaconLocation(use.getBeaconLocation())
+      .windfarmLocation(use.getWindfarmLocation())
```
Comment (Evie-Skinner18) [inline service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java:169]: Just checking do we need to update the unit tests for ExportMapper?

File: service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java [hunk 6]
```diff
@@ -97,4 +98,34 @@ public void test_getMultipleValuesAsString_expectTrimmedValues() {
     //then
     Assert.assertEquals("test / me / please", output);
   }
+
+  @Test
+  public void test_getActivityCARGO_AIRPLANEAsString_expectReadableValue() {
+    //given
```
Comment (Evie-Skinner18) [inline service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java:104]: Beautiful

File: service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java [hunk 7]
```diff
@@ -97,4 +98,34 @@ public void test_getMultipleValuesAsString_expectTrimmedValues() {
...
+    Assert.assertEquals("Cargo Airplane", activityName);
+  }
+
+  @Test
+  public void test_getActivityHOT_AIR_BALLOONAsString_expectReadableValue() {
+    //given
+    Activity activity = Activity.HOT_AIR_BALLOON;
+
+    String activityName = BeaconsStringUtils.enumAsString(activity);
+    //then
```
Comment (Evie-Skinner18) [inline service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java:118]: Consider removing comments. What do these add that's not already communicated by your code?

**Review by Evie-Skinner18 — APPROVED** [review]

File: backoffice/src/views/exports/certificates/Certificate.tsx (hunk 1 above)
Comment (slkendell-mt) [reply backoffice/src/views/exports/certificates/Certificate.tsx:211]: Nice spot!

File: service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java (hunk 5 above)
Comment (slkendell-mt) [reply service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java:169]: Yeah that's next on my list

File: service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java (hunk 7 above)
Comment (slkendell-mt) [reply service/src/test/java/uk/gov/mca/beacons/api/utils/BeaconsStringUtilsUnitTest.java:118]: Fair point, I think it's a habit to enforce separation. 

If more lines were to come, it makes it clear of the cause -> effect of the test

**Review by Evie-Skinner18 — APPROVED** [review]

**Pull request merged**